sim/battle.ts

```diff
@@ -2902,9 +2915,9 @@ export class Battle extends Dex.ModdedDex {
 	 * Takes a choice string passed from the client. Starts the next
 	 * turn if all required choices have been made.
 	 */
-	choose(sideid: string, input: string) {
+	choose(sideid: SideID, input: string) {
 		let side = null;
-		if (sideid === 'p1' || sideid === 'p2') side = this[sideid];
+		if (['p1', 'p2', 'p3', 'p4'].includes(sideid)) side = this.sides[parseInt(sideid[1], 10) - 1];
```

Comment:
I forgot `indexOf` exists lol

Reply:
```suggestion
		side = this.sides[parseInt(sideid[1], 10) - 1];
```

You don't need the `includes` if you're going to do it this way.

Reply:
Although maybe put this in a `getSide` function for reuse.

Reply:
Yeah a function sounds good.